src/qt/governancelist.cpp

```diff
@@ -387,14 +387,12 @@ void GovernanceList::showProposalContextMenu(const QPoint& pos)
         return;
     }
 
+    // right click menu with option to open proposal url
     QString proposal_url = proposal->url();
     proposal_url.replace(QChar('&'), QString("&&"));
 
-    // right click menu with option to open proposal url
-    QAction* openProposalUrl = new QAction(proposal_url, this);
     proposalContextMenu->clear();
-    proposalContextMenu->addAction(openProposalUrl);
-    connect(openProposalUrl, &QAction::triggered, proposal, &Proposal::openUrl);
+    proposalContextMenu->addAction(proposal_url, proposal, &Proposal::openUrl);
```

Comment:
please edit the commit message for 1cdd9fbdf5c0b732236d3b2dc164b2c7a6b1b188 to include justification / why

Reply:
same as https://github.com/bitcoin-core/gui/pull/263

> 2. replaces 3 lines of code with the only call of [`QMenu::addAction`](https://doc.qt.io/qt-5/qmenu.html#addAction-5) for each context menu item (it became possible since https://github.com/bitcoin/bitcoin/pull/21286 was merged)